srcpkgs/bitcoin/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'bitcoin'
 pkgname=bitcoin
-version=0.19.1
-revision=3
+version=0.20.1
+revision=1
 build_style=gnu-configure
 configure_args="--with-incompatible-bdb --disable-ccache --disable-static
  --enable-hardening --with-boost=${XBPS_CROSS_BASE}/usr"
-hostmakedepends="pkg-config yasm protobuf"
+hostmakedepends="pkg-config yasm protobuf autoconf automake libtool"
```

Comment:
those are already implicitly required by setting the `build_style` to `gnu-configure`

Reply:
It's already set as gnu-configure and it doesn't pull them

Reply:
Those are additional, yeah. Autohell welcomes you :)